finance.md

```diff
@@ -0,0 +1,69 @@
+# Financial Accounts and Practices
+
+_Note: Note all of the following have been implemented yet, the Finance WG is working on it_
+
+This document outlines finance and accounting practices of Hypha Worker Co-operative, including:
+
+- Listing of accounts at our financial institutions
```

Comment:
Should we list this in https://link.hypha.coop/inventory too? (They're heavier old-world platforms, but in some ways Desjardins feels no different than Google Docs or Twitter or HubSpot...)

Happy to link out to this doc for specific details though! Should be here, just like we wouldn't keep bitcoin addresses in our hackmd ;)

Reply:
Since CRA is on there I think Desjardins (and Transferwise + QB when we have them) should be on there too. I have added Desjardins and linked.